will/requirements/slack.txt

```diff
@@ -1,4 +1,4 @@
 -r base.txt
-slackclient>=1.2.1,<1.3.0
-markdownify==0.4.1
-
+slack-sdk==3.0.0
+markdownify==0.5.3
+aiohttp
```

Comment:
I don't see the added use of aiohttp in the PR so let's remove it from the requirements file.